app/assets/javascripts/components/CartedProducts.jsx

```diff
@@ -10,27 +10,32 @@ var CartedProducts = React.createClass({
   },
   updateQuantity: function(val, id) {
     console.log(val, "from parent && id: ", id);
-    this.props.carted_products.map((carted_product) =>
-      (carted_product.id == id) ? (
-        carted_product.quantity = val
-      ) : (
-        carted_product
-      )
-    );
+    var that = this;
     $.ajax({
       type: "PATCH",
       url: "/api/carted_products/" + id + "/" + val,
       contentType: "application/json; charset=utf-8",
       dataType: "json",
       success: function(result){
-        console.log(result);
+        if (result.message) {
+          console.log("update unsuccessful");
```

Comment:
@heydanhey Figured we'd do something along those lines, but wanted to leave it for now until we decide what would be best.